.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
```suggestion
<!-- Please delete options that are not relevant. -->
```
Sorry, I forgot to say this before in the chat!! Could we make this into a comment instead, in case someone forgets to delete this bit, it wont show up in the PR!

Reply:
As per;
2833d281369dcf1095f5a700d8988f956b95c0fd

This should now be commented out!